source/_docs/z-wave.markdown

```diff
@@ -180,8 +180,25 @@ automation:
         scene_id: 11
 ```
 
+Some devices (like the HomeSeer wall switches) allow you to do things like double, and triple click the up and down buttons and fire an event.  These devices will also send `scene_data` to differentiate the events.  This is an example of double clicking the on/up button:
+
+```yaml
+# Example configuration.yaml automation entry
+automation
+  - alias: 'Dining room dimmer - double tap up'
+    trigger:
+      - event_type: zwave.scene_activated
+        platform: event
+        event_data:
+          entity_id: light.diningroomcans_level
```

Comment:
@AddoSolutions - The only change I'd recommend is that the entity_id here is sent as the zwave entity, not the value entity.  For example, my HomeSeer dimmer will send the event from ```zwave.kitchen_lights``` not ```light.kitchen_lights_level```.